src/main/java/frc/robot/commands/Drive.java

```diff
@@ -34,23 +50,45 @@ public Drive(Drivetrain subDrivetrain, DoubleSupplier xAxis, DoubleSupplier yAxi
 
   @Override
   public void initialize() {
+    northYaw = constField.isRedAlliance() ? Units.Degrees.of(180) : Units.Degrees.of(0);
+    redAllianceMultiplier = constField.isRedAlliance() ? -1 : 1;
```

Comment:
What is this for?

Reply:
The direction that is up/north changes depending on the alliance, so we need to multiply some values by -1 which is established here
Unrelated, but we should probably put all of this in the swerve template as well + an example auto